CRM/Core/I18n.php

```diff
@@ -239,6 +239,26 @@ public static function languages($justEnabled = FALSE) {
     return $justEnabled ? $enabled : $all;
   }
 
+  /**
+   * Get the options available for format locale.
+   *
+   * Note the pseudoconstant can't be used as the key is the name not the value.
+   *
+   * @return array
+   */
+  public static function getLocaleOptions(): array {
+    $values = CRM_Core_OptionValue::getValues(['name' => 'languages'], $optionValues, 'label', TRUE);
+    foreach ($values as $value) {
+      $return[$value['name']] = $value['label'];
```

Comment:
also you seem to be populating an array that hasn't been set here

Reply:
Yeah - that's poor form for IDE friendliness but doesn't enotice - I've fixed it now